src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const thumbs = document.getElementById('thumbs');
+const largeImg = document.getElementById('largeImg');
+
+const NewImage = (events) => {
```

Comment:
Почему функция называется NewImage?
```suggestion
const onImageSelected = (events) => {
```

Reply:
по той причине наверное что она делает большим новое изображение =)

Reply:
1) используй camelCase
2) из названия должно быть очевидно, что это функция